connect/runtime/src/main/java/org/apache/kafka/connect/util/LoggingContext.java

```diff
@@ -0,0 +1,121 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.kafka.connect.util;
+
+import org.slf4j.MDC;
+
+/**
+ * A utility for defining Mapped Diagnostic Context (MDC) for SLF4J logs.
+ */
+public final class LoggingContext {
+
+    /**
+     * The parameter keys used by Connect.
+     */
+    public static final class Parameters {
+
+        /**
+         * The name of the Mapped Diagnostic Context (MDC) key that defines the name of a <i>connector</i>.
+         */
+        public static final String CONNECTOR_NAME = "connector.name";
+
+        /**
+         * The name of the Mapped Diagnostic Context (MDC) key that defines the task number of a <i>connector</i>.
+         */
+        public static final String CONNECTOR_TASK = "connector.task";
+
+        /**
+         * The name of the Mapped Diagnostic Context (MDC) key that defines a <i>scope</i> within a connector.
+         */
+        public static final String CONNECTOR_SCOPE = "connector.scope";
+    }
+
+    /**
+     * The {@link Parameters#CONNECTOR_SCOPE scope} values used by Connect.
+     */
+    public enum Scope {
+        /**
+         * The {@link Parameters#CONNECTOR_SCOPE scope} value for the worker as it starts a connector.
+         */
+        WORKER("worker"),
+
+        /**
+         * The {@link Parameters#CONNECTOR_SCOPE scope} value for {@link org.apache.kafka.connect.connector.Task}
+         * implementations.
+         */
+        TASK("task"),
+
+        /**
+         * The {@link Parameters#CONNECTOR_SCOPE scope} value for committing offsets.
+         */
+        OFFSETS("offsets");
+
+        private final String value;
+        Scope(String value) {
+            this.value = value;
+        }
+        public String value() {
```

Comment:
I've switched to `toString()` which is better anyway since it can be overridden and it should be used when you want a string form of the enum (which we do) as opposed to the name.